This patch is part of a series of patches for festival for
Debian GNU/Linux, which primarily fix build errors with GCC 4.3 and newer.

Index: festival/src/modules/Text/text_modes.cc
===================================================================
--- festival.orig/src/modules/Text/text_modes.cc	2009-02-17 11:45:52.000000000 -0600
+++ festival/src/modules/Text/text_modes.cc	2010-07-17 08:01:24.000000000 -0500
@@ -52,6 +52,9 @@
 #include "text.h"
 #include "lexicon.h"
 
+using std::cerr;
+using std::endl;
+
 static void um_apply_filter(const EST_String &filtername,
 			    const EST_String &infile,
 			    const EST_String &outname);
